.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 **/*.rs.bk
+src/asset.in
+frontend/dist/
```

Comment:
This now means we require `ember` to build `rustdoc`, right?

That's something I was trying to get away from...